test/ng/parseSpec.js

```diff
@@ -2692,6 +2692,15 @@ describe('parser', function() {
           });
         });
 
+       it('should prevent the exploit', function() {
```

Comment:
If you have a better name, then I am ok with it. I do not know if the exploit can be explained in a one line.